plasmapy/physics/quantum.py

```diff
@@ -281,3 +284,157 @@ def Thomas_Fermi_length(n_e):
+    often used to calculate the coupling parameter.
+    When ion density is used, this is the ion sphere radius.
+
+    Parameters
+    ----------
+    n: Quantity
+        Particle number density
+
+    Returns
+    -------
+    radius: Quantity
+        The Wigner-Seitz radius in meters
+
+    Raises
+    ------
+    TypeError
+        If argument is not a Quantity
+
+    UnitConversionError
+        If argument is in incorrect units
+
+    ValueError
+        If argument contains invalid values
+
+    UserWarning
+        If units are not provided and SI units are assumed
+
+    Notes
+    -----
+    The Wigner-Seitz radius approximates the interparticle spacing. 
+    It is the radius of a sphere whose volume is equal to the mean 
+    volume per atom in a solid:
+
+    .. math::
+    r = \left(\frac{3}{4 \pi n}\right)^{1/3}
+
+    See also
+    --------
+    Fermi_energy
+
+    Example
+    -------
+    >>> from astropy import units as u
+    >>> Wigner_Seitz_radius(1e29 * u.m**-3)
+    <Quantity 1.3365046175719775e-10 m>
+
+    """
+    radius = (3 / (4 * np.pi * n)) ** (1 / 3)
+    return radius
+
+
+def chemical_potential(n_e, T, tol=1e-6):
+    r"""Calculate the ideal chemical potential
+
+    Parameters
+    ----------
+    n_e: Quantity
+        Electron number density
+
+    Returns
+    -------
+    radius: Quantity
+        The Wigner-Seitz radius in meters
+
+    Raises
+    ------
+    TypeError
+        If argument is not a Quantity
+
+    UnitConversionError
+        If argument is in incorrect units
+
+    ValueError
+        If argument contains invalid values
+
+    UserWarning
+        If units are not provided and SI units are assumed
+
+    Notes
+    -----
+    The ideal chemical potential is given by [1]:
+
+    .. math::
+    \chi_a = I_{1/2}(\beta \mu_a^{ideal})
+    where \chi is the degeneracy parameter, I_{1/2} is the Fermi integral
+    with order 1/2, \beta is the inverse thermal energy \beta = 1/(k_B T),
+    and \mu_a^{ideal} is the ideal chemical potential.
+
+    See also
+    --------
+
+    References
+    ----------
+    [1] Bonitz, Michael. Quantum kinetic theory. Stuttgart: Teubner, 1998.
+
+    Example
+    -------
+    >>> from astropy import units as u
+    >>> Wigner_Seitz_radius(1e29 * u.m**-3)
+    <Quantity 1.3365046175719775e-10 m>
+    """
+    # deBroglie wavelength
+    lambdaDB = thermal_deBroglie_wavelength(T)
+    # degeneracy parameter
+    degen = n_e * lambdaDB ** 3
+
+    def minFunc(alpha):
+        """Function to be minimized"""
+        # Note: alpha = mu / (k_B * T)
+        zeroVal = Fermi_integral(alpha, 1 / 2) - degen
+        return zeroVal
+    # optmizing minFunc to numerically obtain the ideal chemical potential
+    #!!! implement approximation for initial guess!!!
+    # https://de.wikipedia.org/wiki/Fermi-Dirac-Integral
+    alphaGuess = k_B * T
+    alphaFinal = minimize(minFunc,
+                          alphaGuess,
+                          method='Nelder-Mead',
```

Comment:
Leave a note saying why this particular method!

Reply:
This is arbitrary and by no means fixed. Needs testing.